src/imitation/algorithms/adversarial.py

```diff
@@ -352,10 +395,28 @@ def init_trainer(env_name: str,
     init_rl_kwargs: Keyword arguments passed to `init_rl`,
         used to initialize the RL algorithm.
   """
+  # come up with an informative, unique name for this expt
+  log_prefix = log_dir if log_dir is not None else 'output'
+  date_str = datetime.datetime.now().strftime('%Y-%m-%dT%H:%M')
+  method_name = 'gail' if use_gail else 'airl'
+  log_base = os.path.join(
+    log_prefix, f'{method_name}_{env_name}_{date_str}')
```

Comment:
Sorry, didn't realise. I wasn't using `scripts.train_adversarial()` because I wanted to control the training loop, hence the date string here. It would be good to move all the unique experiment identifier generation stuff to one place, though (ideally up the stack from this code, or at least in a place where it can be used by all algorithms).

Reply:
Fixed by forcing caller to pass in `log_dir` in full.